jep/229/README.adoc

```diff
@@ -88,41 +88,34 @@ after a link:https://ci.jenkins.io/job/Plugins/job/log-cli-plugin/job/master/21/
-
-* Use the Action for Release Drafter to create a release (typically including exactly one entry, from a recently merged PR).
-* Create a new Action wrapping the `release.sh` + `settings.xml` so we do not have this stuff duplicated.
-* Add `block-MRP` profile to `plugin-pom`.
+Release Drafter is configured as an Action as well, but always publishing the draft release under the placeholder name `next` rather than a real version.
+This runs (fairly) quickly after PRs are merged, and just updates a draft release.
+If and when a release is successfully deployed to Artifactory, the deployment script
+link:https://github.com/jglick/jenkins-maven-cd-action/blob/3f1f2fa2739ce0747de9a2545aee5d54917a7abb/run.sh#L5-L8[creates a (lightweight) Git tag and publishes the release].
 
 === Organization setup
 
 To avoid the need for each component maintainer to manually create a bot account and perform similar setup,
-tooling at the organization level should automate this aspect.
-
-link:https://github.com/jenkins-infra/repository-permissions-updater/blob/6ff61dc11b830f1984dde4ba9e82870218d702f7/Jenkinsfile#L54-L56[Repository Permissions Updater (RPU) processing]
-is patched to recognize some flag in a repo’s YAML entry (e.g., `bot: true`)
-and link:https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-CreateorReplaceUser[create an Artifactory bot account] with permissions to only the listed paths,
-then link:https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-CreateToken[generate an access token for that account]
-and upload it to the GitHub secret `ARTIFACTORY_TOKEN` in the corresponding repository.
-(Maybe also create an `ARTIFACTORY_USER` pseudo-secret to match.)
+tooling at the organization level automates this aspect.
 
-It is possible to upload artifacts using a token with a short expiry (say, an hour), and a phony user name.
-The problem is that it is not possible to assign permissions from a “permission target” to an access token, just a group membership.
-RPU will need a larger overhaul for this, between generating/maintaining groups;
-poking GitHub’s Secrets API to update the deployment secret to be valid for some period (such as the RPU run interval + 3 hours or so);
-and perhaps changing the YAML file format to accommodate multi-module projects and the new automated builds.
+link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permissions Updater (RPU) processing]
+recognizes a flag in a repo’s YAML entry (`cd.enabled` set to `true`).
+If that flag is set, a group whose name is derived from the GitHub repository name is created in Artifactory.
+That group is granted the same permissions as regular maintainers, but contains no users.
+Instead, RPU periodically generates a temporary Artifactory access token that's a member of that group
+and saves it to the GitHub secrets `ARTIFACTORY_USERNAME` and `ARTIFACTORY_TOKEN` in the corresponding repository.
```

Comment:
Would it make sense to be more implementation agnostic here and call these `MAVEN_REPO_USERNAME` / `MAVEN_REPO_TOKEN` instead?

Reply:
Fine with me. If you want to make such a change in RPU and also push fresh secrets with those names to `log-cli-plugin`, I will update everything to match.